tests/plugins/test_rolecasting_native_codex.py

```
@@ -134,6 +166,54 @@ def test_profile_rejects_executor_drift_before_native_spawn(self) -> None:
+        native = load_module()
+
+        denied = intent()
+        denied["model_transition"] = {
+            **denied["model_transition"],
+            "authorization": {
+                "status": "denied",
+                "reason": "route-capacity-unavailable",
+                "disposition": "blocked",
+            },
+        }
+        with self.assertRaisesRegex(
```

Ruff (0.16.2)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)